gorb/gorb_metrics.go

```diff
@@ -0,0 +1,24 @@
+package gorb
```

Comment:
It doesn't look like this is used at the moment ... am I right?

Reply:
Used line 59 of gorb/gorb.go

Reply:
I can't see it being used either.